tests/phpunit/tests/media.php

```diff
@@ -2286,6 +2286,9 @@ public function test_wp_filter_content_tags_srcset_sizes() {
 	 *
 	 * @ticket 34898
 	 * @ticket 33641
+	 *
+	 * @expectedDeprecated wp_img_tag_add_loading_attr
+	 * @expectedDeprecated wp_get_loading_attr_default
```

Comment:
Please remove all of these annotations throughout the PR on tests that do not specifically test `wp_img_tag_add_loading_attr()` or `wp_get_loading_attr_default()`.

For any _general_ tests, e.g. those that use `wp_filter_content_tags()`, we will need to make sure they still work or modify the tests as needed.

The `@expectedDeprecated` annotations should only be present on tests that specifically test one of the two deprecated functions, i.e. the `test_wp_img_tag_add_loading_attr*` and `test_wp_get_loading_attr_default*` methods.

Reply:
@kt-12 There are still a few tests for `wp_filter_content_tags()` where you haven't removed those annotations. None of them must have those, instead we have to fix the tests to not use deprecated functions anymore.